member_go.py

```
@@ -0,0 +1,105 @@
+             offset_date=last_date,
+             offset_id=0,
+             offset_peer=InputPeerEmpty(),
+             limit=chunk_size,
+             hash = 0
+         ))
+chats.extend(result.chats)
+
+
+for chat in chats:
+    try:
+        if chat.megagroup== True:
```

Ruff (0.8.2)

Avoid equality comparisons to `True`; use `if chat.megagroup:` for truth checks

Replace with `chat.megagroup`

(E712)